src/runtime_src/tools/xclbinutil/unittests/FixedKernel/embedded_metadata_expected.xml

```diff
@@ -67,7 +67,7 @@
         <connection srcType="core" srcInst="OCL_REGION_0" srcPort="noc_32_0_M13_AXI" dstType="kernel" dstInst="vadd_1" dstPort="S_AXI_CONTROL"/>
         <connection srcType="core" srcInst="OCL_REGION_0" srcPort="noc_64_0_S02_AXI" dstType="kernel" dstInst="vadd_1" dstPort="M_AXI_GMEM"/>
         <kernel name="DPU" language="c" type="dpu">
-          <extended-data functional="0" dpu_kernel_id="0x101"/>
+          <extended-data functional="DPU" subtype="DPU" dpu_kernel_id="0x101"/>
```

Comment:
Unfortunately, the XML needs to be numeric and not string values.   This change will break existing customer solutions.

Reply:
add code to convert string to numeric for embedded_metadata xml